### ISSUE TITLE
Error: provider 'gopass' does not exist%

### ISSUE THREAD

**Issue by patrickc-sb** [SpectralOps/teller#87]
## Expected Behavior
Accessing secrets stored in gopass password-store

## Current Behavior
.teller.yaml provider configuration for gopass always throws:
`Error: provider 'gopass' does not exist%`

Provider config for gopass:
```
#
# Providers
#
providers:
  gopass:
    env_sync:
      path: /path/to/gopass/password-store
``` 

Gopass is also installed and working on my MacBook.

I'm not sure what I'm missing here. I'd be thankful for any pointers.

Thanks in advance!

## Steps to Reproduce

1. `teller new`
2. Add gopass configuration to .teller.yaml
3. `teller show`
4. Above mentioned error appears

## Context
I'd like to use gopass as a secret backend.

## Specifications
  - Version: `$ teller version` => 1.4.0
  - Platform: masOS

**Comment by kaplanelad** [comment]
Hey @patrickc-sb, thanks for submitting this issue.
The `gopass` provider is not released under 1.4.0 Teller version. we will release a new Teller version with `gopass` provider and let you know soon.
@jondot, for releasing a new version we need to add a `GORELEASER_GITHUB_TOKEN`, can you please assist?

**Comment by patrickc-sb** [comment]
Hey @kaplanelad,

thanks for your quick response! Perfect, I was a little confused and didn't know if I was forgetting something. Great, that's all I need to know. Then I'll wait until the new version is released.

Thanks so much!

**Comment by kaplanelad** [comment]
Hey @patrickc-sb, 
a new version was released 1.5.0, please upgrade and gopass provider will be ready to use

**Comment by patrickc-sb** [comment]
Oh man, that was fast! Awesome!
I'll check it out, thanks so much!
